### YamlMime:UniversalReference
items:
  - uid: '@azure/servicefabric.TestErrorChaosEvent'
    name: TestErrorChaosEvent
    fullName: TestErrorChaosEvent
    children:
      - '@azure/servicefabric.TestErrorChaosEvent.kind'
      - '@azure/servicefabric.TestErrorChaosEvent.reason'
      - '@azure/servicefabric.TestErrorChaosEvent.timeStampUtc'
    langs:
      - typeScript
    type: interface
    summary: |-
      <span data-ttu-id="92252-101">Descrive un evento di Chaos viene generato quando si verifica un evento imprevisto nel motore di Chaos.</span><span class="sxs-lookup"><span data-stu-id="92252-101">Describes a Chaos event that gets generated when an unexpected event occurs in the Chaos engine.</span></span>
      <span data-ttu-id="92252-102">Ad esempio, a causa dello snapshot di cluster in corso incoerente, durante un'entità di generare un errore, Chaos individuato che l'entità è stata già verificati errori, che potrebbe essere un evento imprevisto.</span><span class="sxs-lookup"><span data-stu-id="92252-102">For example, due to the cluster snapshot being inconsistent, while faulting an entity, Chaos found that the entity was already faulted -- which would be an unexpected event.</span></span>
    package: '@azure/servicefabric'
  - uid: '@azure/servicefabric.TestErrorChaosEvent.kind'
    name: kind
    fullName: kind
    children: []
    langs:
      - typeScript
    type: property
    summary: <span data-ttu-id="92252-103">Discriminatore polimorfiche</span><span class="sxs-lookup"><span data-stu-id="92252-103">Polymorphic Discriminator</span></span>
    syntax:
      content: 'kind: "TestError"'
      return:
        type:
          - '"TestError"'
    package: '@azure/servicefabric'
  - uid: '@azure/servicefabric.TestErrorChaosEvent.reason'
    name: reason
    fullName: reason
    children: []
    langs:
      - typeScript
    type: property
    summary: '<span data-ttu-id="92252-104">Descrive il motivo per cui è stato generato TestErrorChaosEvent.</span><span class="sxs-lookup"><span data-stu-id="92252-104">Describes why TestErrorChaosEvent was generated.</span></span> <span data-ttu-id="92252-105">Ad esempio, Chaos tenta di una partizione di errore, ma individua che la partizione non è più a tolleranza di errore, quindi un TestErrorEvent generato con il motivo che informa che la partizione non è a tolleranza di errore.</span><span class="sxs-lookup"><span data-stu-id="92252-105">For example, Chaos tries to fault a partition but finds that the partition is no longer fault tolerant, then a TestErrorEvent gets generated with the reason stating that the partition is not fault tolerant.</span></span>'
    optional: true
    syntax:
      content: 'reason?: undefined | string'
      return:
        type:
          - undefined | string
    package: '@azure/servicefabric'
  - uid: '@azure/servicefabric.TestErrorChaosEvent.timeStampUtc'
    name: timeStampUtc
    fullName: timeStampUtc
    children: []
    langs:
      - typeScript
    type: property
    summary: <span data-ttu-id="92252-106">Il timestamp UTC quando è stato generato questo evento di Chaos.</span><span class="sxs-lookup"><span data-stu-id="92252-106">The UTC timestamp when this Chaos event was generated.</span></span>
    syntax:
      content: 'timeStampUtc: Date'
      return:
        type:
          - Date
    package: '@azure/servicefabric'